### PR TITLE
Adding R_earth and R_jup to units and docs

### PR DESCRIPTION
**Pull request by bsipocz**
This is to address #4813.  

The logic follows #3907 that added M_earth and M_jup to units.

### CODE REVIEW

**Comment by pllim** [comment]
:+1:

File: CHANGES.rst [hunk 1]
```diff
@@ -105,6 +105,8 @@ New Features
     deprecated in 0.3.1, has been removed. [#4449]
 
   - Added slug to imperial units. [#4670]
+  - Added Earth radius (``R_earth``) and Jupiter radius (``R_jup``) to units.
```
Comment (astrofrog) [inline CHANGES.rst]: Add a whitespace above

**Comment by astrofrog** [comment]
👍  aside from my comment above

**Comment by bsipocz** [comment]
@astrofrog - Done.

**Comment by astrofrog** [comment]
@mhvk - I've assigned this to you, since you are a maintainer of astropy.units

**Comment by mhvk** [comment]
Looks good!  Merging...

**Pull request merged**